### ISSUE TITLE
some formula not work properly in chrome

### ISSUE THREAD

**Issue by niyanchun** [mathjax/MathJax#2069]
Some formula not work properly in Chrome:

formula 1:  
```
$$
{
p(y_i) = p_i^{y_i}(1-p_i)^{1-y_i}
}
$$
```
display：

![wx20181010-235334](https://user-images.githubusercontent.com/5145759/46749195-ce923480-cce7-11e8-8ca3-13e1c2c372fc.png)

minus is missing. 

formula 2:

```
$$
{     
L(\theta)=\prod_{i=1}^n p_i^{y_i}(1 - p_i)^{1-y_i}
}
$$
```

display:

![wx20181010-235626](https://user-images.githubusercontent.com/5145759/46749365-22048280-cce8-11e8-801e-56cd9a01b736.png)

minus and prod are all missing.

These problems happen on chrome,  but work properly on safari.

PS:

OS version:  macOS High Sierra Version 10.13.6
Chrome version: Version 69.0.3497.100 (Official Build) (64-bit)
Safari version: Version 12.0 (13606.2.11)
MathJax version: v2.7.4

And  these formulas can also been found on my blog: https://time-track.cn/logistic-regression-introduction.html

**Comment by dpvc** [comment]
From your images, it looks like MathJax is able to change the font being used in your browser.  (None of the characters are in the correct font, and the missing characters are ones that aren't in your default font.)  Do you have some plugin that disables page-specified fonts?  Can you try running Chrome without plugins and see if that affects anything?

**Comment by niyanchun** [comment]
Thanks for suggestion And I disabled all the plugins, but problem  still exists. And I found some error in console:

```
Redirect from 'https://cdn.bootcss.com/mathjax/2.7.4/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff' to 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://time-track.cn' is therefore not allowed access.
```

I don't know this is my own problem or  others.

**Comment by niyanchun** [comment]
I have found the problem: 

I have used a CDN(BootCDN), it use *cdn.bootcss.com* before, but now change to *cdnjs.cloudflare.com*.  It seems they do a rediect from the old url to new, but there are some CORS problem.  And now I use the new domain in my code, and my display problem solved.

**Comment by phymalidoust** [comment]
Hi,
I have written formulas and matrices but it cannot be interpreted by MathJax extension on my chrome. 

The formula is the following and I tried 'equation', 'align', 'split', .... none of them work. MathJax writes everything in one line. How should I resolve it?

<img width="1347" alt="screen shot 2018-10-18 at 17 20 11" src="https://user-images.githubusercontent.com/41995552/47165573-87fd9500-d2fa-11e8-93d9-e9c12eff6b09.png">


$$
\begin{eqnarray}&\color{blue} {\text{FOR}}\;\{\;T=T_{max}:T_{min}\;\}\;\; \\&x_r^P = x_r^1 = \text{random}()\,\rightarrow E^P = E(x_r^P)\\&x_r^N = x_r^2 = \text{random}()\, \rightarrow E^N = E(x_r^N)\\&\Delta E = E^N - E^P\\&\color{red}{\text{IF}}\;\{\;\Delta E <0\;\} \;\;\;(\text{hill climbing})\\&x_r^P =x_r^N\\&\color{red}{\text{ELSEIF}}\;\{\;\exp({{\Delta E}/T})\;> \text{random}()\;\} \;\;\;&(\text{random walk})\\&x_r^P =x_r^N\\&\color{red}{\text{ENDIF}}\\&\color{blue}{\text{ENDFOR}}&\end{eqnarray}
$$

**Comment by dpvc** [comment]
I suspect that your content management system is converting the double backslash (`\\`) to a single backslash (`\`).  Note that in the `eqnarray` that you provided, the lines end in `\` rather than `\\`.  That is why you are getting everything in one long line.

Use the contextual menu (right-click or command-click) on a typeset expression and select "Show Math As -> TeX Commands" and see if the double-backslashes are actually there.

Also, your issue is unrelated to the initial one in this issue tracker, so you should have started a new issue for your problem.  It just confuses things to add onto an old (closed) issue like this.
